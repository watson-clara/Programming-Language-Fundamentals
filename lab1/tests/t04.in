debug off
load tracks.txt
filter title
filter comments
filter punctuation
count tracks
count words
count characters
