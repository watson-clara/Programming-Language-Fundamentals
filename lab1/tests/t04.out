100000
271634
1525111
